lib/client.js

```diff
@@ -28,7 +28,8 @@ class Client {
    *     cert: certFileContent,
    *     key: keyFileContent,
    *     ca: caFileContent
-   *   }
+   *   },
+   *   bearerToken: 'eyJhbGci...dQssw5c'
```

Comment:
What is this default Bearer token value?  Is this for testing>

Reply:
This is part of the JSDoc